docs/Doxyfile_include_for_windows

```diff
@@ -0,0 +1,16 @@
+# This file is intended to be included in the Doxyfile config for Windows.
+# It contains configuration settings that are incompatible with Linux.
+
+# The FILTER_PATTERNS tag can be used to specify filters on a per file pattern
+# basis. Doxygen will compare the file name with each pattern and apply the
+# filter if there is a match. The filters are a list of the form: pattern=filter
+# (like *.cpp=my_cpp_filter). See INPUT_FILTER for further information on how
+# filters are used. If the FILTER_PATTERNS tag is empty or if none of the
+# patterns match the file name, INPUT_FILTER is applied.
+#
+# Note that we currently have to use a patched version of doxypypy.py because of
+# new syntax in python version 3.8.
+# See https://github.com/Feneric/doxypypy/issues/70
+# The patched version is stored in the current directory.
```

Comment:
how long till this is upstreamed? I'd rather not have patches-in-progress for dependencies commited directly here. Is it a big problem to say that Windows & Py3.8 are currently unsupported? When Fixed upstream, it'll automatically start to work

Reply:
@dkeeney what do you think about this situation at Windows? So far I'm thinking I'd rather use upstream library, and if it does not work for Win..we can generate the docs and publish as part of the CI build anyway?

Reply:
The upstream library does not work in Linux either. The problem is Python, not Windows. Python evolves, but doxypypy has not kept up with the evolution.

Reply:
If one specifies that Python 3.7 must be used, and not some later version, then the upstream library can be used.

Reply:
Lets not lock ourselves into a fixed version of Python.
Is there some other document generator that we could switch to?

Reply:
With doxygen there is at least one alternative for Python, but it is supposed to be inferior. I will have a look.
There is also Sphinx, which is an alternative to doxygen, and supports both C++ and Python. That one is used in docs/Makefile.
My initial results were better with doxygen, but I will take another look.

Reply:
> There is also Sphinx, which is an alternative to doxygen, and supports both C++ and Python 

if it's not too complicated that could be a good choice, given our c++ codebase.